extra_data/file_access.py

```diff
@@ -146,6 +146,8 @@ def __init__(self, filename, _cache_info=None):
             self.train_ids = _cache_info['train_ids']
             self.control_sources = _cache_info['control_sources']
             self.instrument_sources = _cache_info['instrument_sources']
+            self.reduction_data = _cache_info['reduction_data']
```

Comment:
This should either not be cached at all or use conditional access like `validity_flag` further below. Since it's only auxiliary information and may often be comparably small, it may be fine to always generate it on demand. If this changes in the future, we can add machinery to augment existing indices, or simply re-create them.

Reply:
Makes sense. I'll remove from caching.